recipes/spylon/meta.yaml

```diff
@@ -25,6 +25,7 @@ requirements:
     - html5lib
     - lxml
     - beautifulsoup4
+    - ca-certificates
```

Comment:
Would love to know the history behind this addition. Maybe there is something flaky on our end that we need to be aware of.